k8s/shiny-sync

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
So this is just a temporary thing until we have the management bit working?

Reply:
yup temporary until we sort the management side